docs/examples/single_moving_target_relative_Doppler.ipynb

```diff
@@ -0,0 +1,216 @@
+{
```

Comment:
<p>The radar class will support a doppler_processing function where these calculations will be carried out. Here the command can therefore be reduced to <code>radar.doppler_processing()</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rf-eng/scikit-radar/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>